Sources/MarshaledObject.swift

```diff
@@ -22,9 +22,32 @@ public protocol MarshaledObject {
 public extension MarshaledObject {
     public func any(for key: KeyType) throws -> Any {
         let pathComponents = key.stringValue.split(separator: ".").map(String.init)
+	
+        //	Re-combine components ending in backslash…
+	
+		var finalComponents = [String]()
+		var comp = ""
+		for component in pathComponents {
+			if component.hasSuffix("\\") {
```

Comment:
Looks like you're using tabs instead of spaces...

Reply:
Well, of course!

Anyway, changed it.